stdlib/2and3/imaplib.pyi

```diff
@@ -6,15 +6,15 @@ import sys
 import time
 from socket import socket as _socket
 from ssl import SSLSocket
-from typing import Any, Callable, Dict, IO, List, Optional, Pattern, Text, Tuple, Union
+from typing import Any, Callable, Dict, IO, List, Optional, Pattern, Text, Tuple, Type, Union
 
 CommandResults = Tuple[str, List[Any]]
 
 
 class IMAP4:
-    error: Exception
-    abort: Exception
-    readonly: Exception
+    error: Type[Exception] = ...
```

Comment:
Does it work you follow the implementation more closely and write `class error(Exception): ...` here?

Reply:
I tried, but that broke pytype_test.py. Error below:

```
stdlib/2and3/imaplib.pyi: ParseError: syntax error, unexpected CLASS, expecting DEF or '@'
```

Reply:
Oh well. In that case, the current code should be OK.